### PR TITLE
Add data populator info under alpha features

### PR DESCRIPTION
**Pull request by anupriya0703**
Signed-off-by: anupriya0703 <anupriya.gupta@mayadata.io>

### CODE REVIEW

File: docs/main/additional-info/alphafeatures.md [hunk 1]
```diff
@@ -45,3 +46,162 @@ For additional details and detailed instructions on how to get started with Open
...
+1. To get started, the data populator CRD needs to be installed on the setup. To install, execute:
+
+ ```
+ kubectl apply -f https://raw.githubusercontent.com/openebs/data-populator/master/deploy/crds/openebs.io_datapopulators.yaml
+ ```
+2. Next, to install the data populator controller, execute:
+
+ ```
+ kubectl apply -f https://raw.githubusercontent.com/openebs/data-populator/master/deploy/yamls/data-populator.yaml
+ ```
```
Comment (Ab-hishek) [inline docs/main/additional-info/alphafeatures.md:76]: The two steps here can consolidated into one step. Checkout the latest doc here -> https://github.com/openebs/data-populator/blob/develop/docs/data-populator/data-populator.md

File: docs/main/additional-info/alphafeatures.md [hunk 2]
```diff
@@ -45,3 +46,162 @@ For additional details and detailed instructions on how to get started with Open
...
+:::tip NOTE
+  The `openebs-data-population` namespace is reserved for populator and no PVC with `dataSourceRef` should be created in this namespace as the data populator controller ignores the PVCs in its working namespace.
+:::
+
+### Using Data Populator
+
+This section demonstrates the application of data populator. We will be preparing a source for the data populator with some data and then scale down the application. Next, we will be creating an instance of DataPopulator CR. We will then use the new PVC in the deployment spec of the application.
```
Comment (Ab-hishek) [inline docs/main/additional-info/alphafeatures.md:83]: You also add a statement here stating "if you already have a source volume handy, you can simply scale down the application if any uses that source volume" and the rest steps remain the same. Only the preparing the source thing should be omitted in this case i.e the flow can be as follows:
step 1 -> step 4 to step 7

Does this change seem okay to you? @anupriya0703

Comment (anupriya0703) [reply docs/main/additional-info/alphafeatures.md:83]: I was thinking, as it is more like a demonstration of "how to use" we might not need to add this info. Please let me know if you have a deferred opinion.

Comment (Ab-hishek) [reply docs/main/additional-info/alphafeatures.md:79]: What do you think about this @pawanpraka1?

Comment (pawanpraka1) [reply docs/main/additional-info/alphafeatures.md:79]: that is on as it is demostrating a particular use case.

**Comment by pawanpraka1** [comment]
@anupriya0703 instead of copy pasing the content, can we just refer to the doc in repo.

**Review by pawanpraka1 — APPROVED** [review]

**Review by Ab-hishek — APPROVED** [review]

**Pull request merged**